cmd/check.go

```diff
@@ -128,10 +128,12 @@ func runCheckCmd(cmd *cobra.Command, args []string) {
 
 func validateOutputFormat() error {
 	outputFormat := viper.GetString(constants.ArgOutput)
-	if !helpers.StringSliceContains([]string{"text", "brief", "json", "none"}, outputFormat) {
-		return fmt.Errorf("invalid output format '%s' - must be one of json, text, brief, none", outputFormat)
+	// try to get a formatter for the desired output.
+	if _, err := controldisplay.GetFormatter(outputFormat); err != nil {
```

Comment:
what error is shown for an invalid output arg? - screenshot?

Reply:
![Screenshot 2021-06-07 at 7 55 53 PM](https://user-images.githubusercontent.com/1114038/121034156-6cf13380-c7ca-11eb-903c-718cd005a930.png)